src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -415,6 +416,13 @@ private long CreateBackupTask(PeriodicBackup periodicBackup, bool isFullBackup)
                         $"The task cannot run because the CPU credits allocated to this machine are nearing exhaustion.");
                 }
 
+                if (LowMemoryNotification.Instance.LowMemoryState)
+                {
+                    throw new MaxNumberOfConcurrentBackupsException(
```

Comment:
I think it would be better to have a dedicated exception that is caught in `StartBackupTaskAndRescheduleIfNeeded` and handled correctly e.g. we might not want to schedule new backup in 1min after having this

Reply:
Done